frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteRepositoryUpdateBackgroundable.java

```diff
@@ -94,7 +94,7 @@ public void run(ProgressIndicator indicator) {
     }
   }
 
-  private IBranchLayout createBranchLayout(Path path) throws MacheteFileReaderException {
+  private IBranchLayout readBranchLayout(Path path) throws MacheteFileReaderException {
     return Try.of(() -> branchLayoutReader.read(path))
```

Comment:
TBH while you're here, I'd use the good ol' `try...catch` here, I'd be more readable (and no downcast to `BranchLayoutException` will be necessary)